nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/layers/recurrent/weights/RNNWeights.java

```diff
@@ -32,4 +33,8 @@
     public SDVariable[] argsWithInputs(SDVariable... inputs){
         return ArrayUtil.combine(inputs, args());
     }
+
+    public SDVariable[] argsWithInputs(INDArray... inputs) { return (SDVariable[]) ArrayUtil.combine(inputs, args()); };
```

Comment:
You can't do this - SDVariable is symbolic, INDArray is an actual array.
We'll need bigger changes to RNNWeights (and its subclasses) to handle INDArrays